test/queue_tests.py

```diff
@@ -10,17 +10,17 @@
 
 class WhenDeclaringAQueue(OpenChannelContext):
     def when_I_declare_a_queue(self):
-        self.async_partial(self.channel.declare_queue('my.nice.queue', durable=True, exclusive=True, auto_delete=True))
+        self.async_partial(self.channel.declare_queue('my.nice.queue', durable=True, exclusive=True, auto_delete=True, arguments={'x-expires': 300, 'x-message-ttl': 1000}))
 
     def it_should_send_a_QueueDeclare_method(self):
-        expected_method = spec.QueueDeclare(0, 'my.nice.queue', False, True, True, True, False, {})
+        expected_method = spec.QueueDeclare(0, 'my.nice.queue', False, True, True, True, False, {'x-expires': 300, 'x-message-ttl': 1000})
```

Comment:
Perhaps this functionality would also benefit from an end-to-end integration test which uses an actual Rabbit server. What do you think?